mail/opensmtpd/Portfile

```
@@ -0,0 +1,114 @@
+                    \n  Packet Filter's (PF) \
+                    \n* fast and efficient: it can handle large queues with reasonable performance
+homepage            https://www.opensmtpd.org
+
+github.tarball_from archive
+
+checksums           rmd160  186ee26a72f63aa3058bf9990a5a9db12f8ac2ae \
+                    sha256  e35a2954dad002ffaf2060358aa97e5268dbe653e22cdd5ebf7addb1dcabdcdb \
+                    size    447879
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:bison \
+                    port:libtool
+
+depends_lib         port:libevent \
+                    port:openssl
+
+startupitem.create  yes
+startupitem.name    OpenSMTPD
+startupitem.autostart   yes
+startupitem.executable  ${prefix}/sbin/smtpd
+
+add_users           _smtpd group=_smtpd realname=OpenSMTP\ Daemon \
+                    _smtpq group=_smtpq realname=OpenSMTP\ Queue
+
+set sysconfdir ${prefix}/etc/${name}
+
+post-extract {
+    reinplace "s|file:/etc/mail/aliases|file:${sysconfdir}/aliases|g" ${worksrcpath}/usr.sbin/smtpd/smtpd.conf
+}
+
+#use_autoreconf      yes
+#autoreconf.cmd      ./bootstrap
+
+pre-configure {
+    system -W ${worksrcpath} "./bootstrap"
+}
+
+configure.args      --with-auth-pam \
+                    --with-libevent=${prefix} \
+                    --sysconfdir=${sysconfdir} \
+                    --with-path-queue=${prefix}/var/spool/smtpd \
+                    --with-table-db
+
+destroot.keepdirs   ${destroot}${prefix}/var/spool/smtpd
+
+pre-destroot {
+    addgroup _smtpd
+    addgroup _smtpq
+}
+
+post-destroot {
+    xinstall -o root -g wheel -m 711 -d ${destroot}${prefix}/var/spool/smtpd
+    file rename ${destroot}${sysconfdir}/smtpd.conf ${destroot}${sysconfdir}/smtpd.conf.dist
+
+    file link -symbolic ${destroot}${prefix}/sbin/sendmail smtpctl
+    file link -symbolic ${destroot}${prefix}/sbin/mailq smtpctl
+    file link -symbolic ${destroot}${prefix}/sbin/makemap smtpctl
+    file link -symbolic ${destroot}${prefix}/sbin/newaliases smtpctl
+}
+
+# fix group creation if installed from the binary package
+post-install {
+    addgroup _smtpd
+    addgroup _smtpq
+}
+
+post-activate {
+    if {![file exists ${sysconfdir}/smtpd.conf]} {
+        file copy ${sysconfdir}/smtpd.conf.dist ${sysconfdir}/smtpd.conf
+    }
+    if {![file exists ${sysconfdir}/aliases] && [file exists /etc/postfix/aliases]} {
+        file copy /etc/postfix/aliases ${sysconfdir}/aliases
+    }
+}
+
+notes "
+    Both OpenSMTPD and the built-in Postfix listen on port 25 by default.
+    As a result, you will need to stop and disable Postfix before OpenSMTPD
+    can run. To stop Postfix, please run:
+
+        sudo /bin/launchctl stop com.apple.postfix.master
+
+    It has become increasingly difficult to disable Postfix. You will need
+    to first boot into recovery mode and temporarily disable the System
```

Allow MacPorts to word wrap this to the user's terminal width.
```suggestion
    Both OpenSMTPD and the built-in Postfix listen on port 25 by default.\
    As a result, you will need to stop and disable Postfix before OpenSMTPD\
    can run. To stop Postfix, please run:

        sudo /bin/launchctl stop com.apple.postfix.master

    It has become increasingly difficult to disable Postfix. You will need\
    to first boot into recovery mode and temporarily disable the System\
```